frontend/packages/dev-console/src/components/pipelineruns/PipelineTaskStatus.tsx

```diff
@@ -0,0 +1,87 @@
+/* eslint-disable no-undef */
+import * as React from 'react';
+import { VictoryStack, VictoryBar } from 'victory';
+import { Tooltip } from '@console/internal/components/utils/tooltip';
+import { getTaskStatus, PipelineRun } from '../../utils/pipeline-augment';
+import '../pipelines/PipelineVisualizationTask.scss';
+
+interface PipelineTaskStatusProps {
+  pipelinerun: PipelineRun;
+}
+
+interface TaskStatusToolTipProps {
+  tooltip: Tooltip;
+}
+
+interface Tooltip {
+  Succeeded?: number;
+  Running?: number;
+  Failed?: number;
+  Notstarted?: number;
+  FailedToStart?: number;
+}
+
+//The colors are not final. pf-colors are not supported in the charts
+export const mapColorMessage = {
+  Succeeded: { color: 'green', message: 'Succeded' },
+  Running: { color: '#0066cc', message: 'Running' },
+  Failed: { color: 'red', message: 'Failed' },
+  Notstarted: { color: '#ccc', message: 'Not Started Yet' },
+  FailedToStart: { color: 'red', message: 'PipelineRun Failed To Start' },
+};
+
+const getMessage = (status: string, value: number = 0): string => {
+  if (status === 'Notstarted' || status === 'FailedToStart') {
+    return mapColorMessage[status].message;
+  }
+  return `${value} ${mapColorMessage[status].message}`;
+};
+
+export const TaskStatusToolTip: React.FC<TaskStatusToolTipProps> = (props) => {
+  const { tooltip } = props;
+  return (
+    <div className="odc__task__status">
+      {Object.keys(tooltip).map((status) => {
+        return (
+          <div className="row" key={status}>
+            <div className="col-lg-1 col-md-1 offset-lg-1 offset-md-1">
+              <div
+                className="tooltip__legend"
+                style={{ background: mapColorMessage[status].color }}
+              />
+            </div>
+            <div className="col-lg-9 col-md-9  offset-lg-1 offset-md-1 tooltip__text">
+              {getMessage(status, tooltip[status])}
+            </div>
+          </div>
+        );
+      })}
+    </div>
+  );
+};
+
+export const PipelineTaskStatus: React.FC<PipelineTaskStatusProps> = (props) => {
+  const { bars, tooltip } = getTaskStatus(props.pipelinerun);
+  return (
+    <div>
+      <Tooltip
+        content={[<TaskStatusToolTip key={Math.floor(100 * Math.random())} tooltip={tooltip} />]}
```

Comment:
React is throwing error 'All children must have keys' unless I add key to '<TaskStatusToolTip />', perhaps this is because of the [] which holds it. It is the convention to provide Component in [] to tooltip in Console

Reply:
Oh i missed that this is an array. Why is it an array? Doesn't look like there's any valid reason for it to be. 
Don't pass content as an array and you won't get the warning.

Reply:
added a static key, It isn't working without the []

Reply:
@abhinavdahiya The array isn't necessary. Please try again. I even pulled your PR and made the local change successfully.

Reply:
:confused: what PR and what list is this about?

Reply:
@abhinavdahiya  Sorry, you were mis-commented into this conversation :)

Reply:
Sorry my mistake!